### PR TITLE
 use library-go methods for webconsole operator

### PR DESCRIPTION
**Pull request by deads2k**
Removes much of the boilerplate for the webconsole operator and replaces it with generic function.

/assign @mfojtik 
/assign @sttts

### CODE REVIEW

File: pkg/cmd/openshift-operators/webconsole-operator/cmd.go [hunk 1]
```diff
@@ -32,69 +22,12 @@ type WebConsoleOperatorCommandOptions struct {
...
-
-			return options.RunWebConsoleOperator()
-		},
-	}
+func NewWebConsoleOperatorCommand(name string) *cobra.Command {
+	cmd := controllercmd.
+		NewControllerCommandConfig("openshift-web-console-operator", version.Get(), RunWebConsoleOperator).
+		NewCommand()
+	cmd.Use = name
+	cmd.Short = "Install the OpenShift webconsole"
```
Comment (mfojtik) [inline pkg/cmd/openshift-operators/webconsole-operator/cmd.go]: nit: install and operate ;-)

File: pkg/cmd/openshift-operators/webconsole-operator/starter.go [hunk 2]
```diff
@@ -40,4 +37,6 @@ func (o *WebConsoleOperatorStarter) Run(stopCh <-chan struct{}) {
 	kubeInformersNamespaced.Start(stopCh)
 
 	operator.Run(1, stopCh)
+
+	return fmt.Errorf("stopped")
```
[the remark this answers is not in the document]
Comment (deads2k) [reply pkg/cmd/openshift-operators/webconsole-operator/starter.go]: > will this now always error out?

That call never returns.  We'd only stop of the lease is lost.  If the lease is lost, it glog.Fatal's as I recall.

**Pull request merged**